src/app/app.module.ts

```diff
@@ -48,6 +48,7 @@ const matomoTrackers = environment.matomoSiteId ? [
       }
     }),
     RouterModule.forRoot(routes, {
+      bindToComponentInputs: true,
```

Comment:
+1
I read https://itnext.io/bind-route-info-to-component-inputs-new-router-feature-1d747e559dc4 , looks like a good feature.